config/locales/views/schools/schools.yml

```diff
@@ -354,7 +355,25 @@ en:
         message: Make a plan for reducing energy use to save money and reduce emissions by focusing on these key areas
       programme:
         choose_a_new_programme_message: It's time to choose a new programme of activities. What challenge will you take on next?
+        join:
+          completed_message_html:
+            one: You've completed all the tasks in the <strong>%{title}</strong> programme. Mark it done to score <strong>%{count}</strong> bonus point?
+            other: You've completed all the tasks in the <strong>%{title}</strong> programme. Mark it done to score <strong>%{count}</strong> bonus points?
+            zero: You've completed all the tasks in the <strong>%{title}</strong> programme. Mark it as complete?
+          message_html:
+            one: You've recently completed a task that is part of the <strong>%{title}</strong> programme. Do you want to enrol in the programme?
+            other: You've recently completed %{count} tasks that are part of the <strong>%{title}</strong> programme. Do you want to enrol in the programme?
         message_html: You have completed <strong>%{programme_activities_count}/%{activity_types_count}</strong> of the activities in the <strong>%{programme_type_title}</strong> programme
+        progress:
+          message_actions_only_html: You have completed <strong>%{completed_actions_count}/%{total_actions_count}</strong> of the actions in the <strong>%{title}</strong> programme
```

Comment:
Seems inconsistent to refer to activities/actions/or both when reporting progress, when the rest of the wording only refers to "tasks". It'd be more consistent to just use "tasks" everywhere I think.

Reply:
Soo... do you mean always refer to them together? So rather than splitting them out as we do now, always combine the two?
Rather than:
`You have completed <strong>%{completed_activities_count}/%{total_activities_count}</strong> of the activities and <strong>%{completed_actions_count}/%{total_actions_count}</strong> of the actions in the <strong>%{title}</strong> programme
`We have this instead:
You have completed <strong>%{completed_tasks_count}/%{total_tasks_count}</strong> of the tasks in the <strong>%{title}</strong> programme

Reply:
Yes. I thought that's what'd decided when you asked the question on Trello?

You had this as the example: ”You've recently completed 6 tasks that are part of the Cut Out the High Carbon Commute programme. Do you want to enrol in the programme?” of the preferred approach I'd read that as an example of how we'd frame all the messages?